index.ts

```diff
@@ -0,0 +1,44 @@
+import { RGBot } from "rg-bot";
```

Comment:
In other templates we have RGBot.RGBot

Reply:
If you're referring to `const RGBot = require("rg-bot").RGBot;` in our js samples, this is intentional